osu.Game.Rulesets.Taiko/Mods/TaikoModPlayfieldResize.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Layout;
+using osu.Game.Beatmaps;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Taiko.Objects;
+using osu.Game.Rulesets.Taiko.UI;
+using osu.Game.Rulesets.UI;
+using osuTK;
+
+namespace osu.Game.Rulesets.Taiko.Mods
+{
+    public class TaikoModPlayfieldResize : Mod, IApplicableToDrawableRuleset<TaikoHitObject>, IApplicableToDifficulty
+    {
+        public override string Name => "Playfield Resize";
+        public override string Acronym => "PR";
+        public override string Description => @"Resize the playfield to see more or less notes";
+        // Ideally, this should go to System and be configurable from settings or something
+        public override ModType Type => ModType.Fun;
+        public override double ScoreMultiplier => 1;
+
+        public override string SettingDescription => PlayfieldScale.IsDefault ? "4:3" : $"Playfield Scale {PlayfieldScale.Value}";
+
+        [SettingSource("Playfield Scale", "The new playfield size")]
```

Comment:
This needs to be explained better. A value between 0.5 and 2 with no units doesn't tell much. Especially so that it's apparently a ratio relative to 16:9 which is extra confusing.

Reply:
Right, this is a poor description